*** spkpvn.c.n0066.original	2017-10-26 16:24:25.331998839 -0400
--- spkpvn.c.n0066.patched	2017-10-26 16:23:36.203367632 -0400
***************
*** 60,65 ****
--- 60,66 ----
  	    integer *, ftnlen);
      extern logical return_(void);
  
+ 
  /* $ Abstract */
  
  /*     Return the state (position and velocity) of a target body */
***************
*** 499,504 ****
--- 500,515 ----
  	chkout_("SPKPVN", (ftnlen)6);
  	return 0;
      }
+ 
+ 
+     {
+     extern btclogspkpvn_ ( integer     * handle
+                          , doublereal  * descr
+                          , doublereal  * et
+                          );
+         btclogspkpvn_ ( handle, descr, et);
+     }
+ 
      chkout_("SPKPVN", (ftnlen)6);
      return 0;
  } /* spkpvn_ */
